tests/unit/test_quorum.py

```diff
@@ -33,18 +33,20 @@ def harness():
 
 
 def test_get_updated_servers(harness):
```

Comment:
**suggestion**: Adding a comment with a rundown of what is going on in this test.

It took me a fair amount of time and a local investigation to understand everything in here

Reply:
Will do :)